clients/cobol-lsp-vscode-extension/snippets.json

```diff
@@ -2085,5 +2085,22 @@
         ],
         "description": "[IDMS] Template Program",
         "scope": "cobol"
+    },
+    "READ TRANSACTION": {
+        "prefix": "READ TRANSACTION",
+        "body": [
+            "READ TRANSACTION ${1:task-name}."
+        ],
+        "description": "[DAF] Read Transaction Statement",
```

Comment:
Should it be "DAF" or "DaCo"?

Reply:
Will confirm with Mark and update accordingly.